src/components/PrivacyStatement/index.js

```diff
@@ -0,0 +1,50 @@
+import { h, Component} from 'preact'
+
+import theme from '../Theme/style.css'
+import style from './style.css'
+import Title from '../Title'
+import {preventDefaultOnClick} from '../utils'
+import {sendScreen} from '../../Tracker'
+
+class PrivacyStatement extends Component {
+  componentDidMount() {
+    sendScreen(['privacy'])
+  }
+
+  render({i18n, back, acceptTerms}) {
+    const title = i18n.t('privacy.title')
+    return (
+      <div className={style.privacy}>
+        <Title {...{title}} />
+        <div className={`${theme.thickWrapper} ${style.content}`}>
+          <ul className={style.list}>
+            <li className={style.item}>{i18n.t('privacy.item_1')}</li>
+            <li className={style.item}>{i18n.t('privacy.item_2')}</li>
+            <li className={style.item}>{i18n.t('privacy.item_3')}</li>
+          </ul>
+
+          <div>
+            <div className={style.smallPrint}>
+              {i18n.t('privacy.small_print_p1')}
+              <a href='https://onfido.com/termofuse' target='_blank'>{i18n.t('privacy.terms_link')}</a>
+              {i18n.t('privacy.small_print_p2')}
+              <a href='https://onfido.com/privacy' target='_blank'>{i18n.t('privacy.privacy_link')}</a>
```

Comment:
This is a really brittle way of doing i18n

You're assuming that any language you want to translate to will fit into the structure you've written here. But you might get a language where it sounds way more natural to order this as "Onfido Terms of Use you are agreeing to" (Japanese is like that, I think)

We're handling this elsewhere by explicitly storing i18n as markdown, and parsing that. But it's fairly heavyweight, we can just get away with it for now. You might want to either make an explicit fragment class, or trust your translations and use `dangerouslySetInnerHtml` through a translation wrapper (as sort of discussed [here](https://groups.google.com/forum/#!topic/reactjs/3b_P6UNjB5s))

But also feel free to just kick the bucket down with this one. It'll definitely bite you at some point though, so bear it in mind

Reply:
That's a very good point. I will have look at fixing this tomorrow.